app/controllers/api/v1/npq_applications_controller.rb

```diff
@@ -20,6 +20,29 @@ def index
         end
       end
 
+      def reject
+        profile = npq_lead_provider.npq_profiles.includes(:user, :npq_course).find(params[:id])
```

Comment:
This would make a good `policy_scope` I expect

Reply:
do we need pundit here? we're just affecting objects the token owns. theres no different access levels of users here or the need to deal with permissions from what I can tell